InvoiceMaker/Controllers/WorkDoneController.cs

```diff
@@ -39,7 +39,7 @@ public ActionResult Create()
         }
 
         [HttpPost]
-        public ActionResult Create(CreateWorkDone model)
+        public ActionResult Create(CreatInvoice model)
```

Comment:
Why use `CreatInvoice` here? Aren't you creating a `WorkDone` record?

Reply:
Whoops! This is copy pasting gone wrong here.